lighthouse-core/test/gather/driver-test.js

```diff
@@ -485,14 +678,19 @@ describe('.gotoURL', () => {
         },
       };
 
-      expect.assertions(2);
+      // 2 assertions in the catch block and the 1 implicit in `findListener`
+      expect.assertions(3);
 
       try {
         const loadPromise = driver.gotoURL(startUrl, loadOptions);
         await flushAllTimersAndMicrotasks();
+
+        const listener = driver.on.findListener('Security.securityStateChanged');
+        listener(insecureSecurityState);
```

Comment:
hmm, why do it this way rather than with `mockEvent()`? (which would have to be modified, obvs)

Or leave a comment giving a heads up on doing it this way to have it fire later

Reply:
mockEvent would respond immediately, we want to control exact flow so we can assert, added a comment